content/microcontrollers/arduino-nano33-iot.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Arduino Nano33 IoT"
+weight: 3
+---
+
+The [Arduino Nano33 IoT](https://www.adafruit.com/product/3333) is a very small ARM development board based on the Atmel [SAMD21](https://www.microchip.com/wwwproducts/en/ATSAMD21G18) family of processors. It also has a NINA-W102 chip onboard which provides an wireless communication abilities based on the popular ESP32 family of wireless chips from Espressif.
```

Comment:
link goes to circuit playground express, I think this is the correct one https://store.arduino.cc/nano-33-iot 
or maybe we should link to the blog post announcing it